src/bin/ion/commands/beta/generate/templates/java/class.templ

```diff
@@ -16,4 +21,110 @@ public final class {{ target_kind_name }} {
         return this.{{ field.name | camel }};
     }
     {% endfor %}
+
+    public static {{ target_kind_name }} readFrom(IonReader reader) {
+        {# Intializes all the fields of this class #}
+        {% for field in fields -%}
+        {{ field.value }} {{ field.name | camel }} =
+            {% if field.value == "boolean" %}
+                false
```

Comment:
If you use `Boolean` instead of `boolean`, then this could be initialized to `null`. Similarly for `int`/`Integer` and `double`/`Double`.

However, if it's not an optional field, then we're better off using the primitive types to avoid the overhead of the boxed types.

Reply:
I am going to keep this as boolean/int/double for now. I will change it in separate PR to consider nulls.

Reply:
Reference: #101